alpheios_nemo_ui/data/assets/scss/breadcumb-navbar.scss

```diff
@@ -0,0 +1,86 @@
+.breadcumb-navbar {
+    overflow: hidden;
+    display: block;
+    text-align: center;
+
+    .nav-link-current-text {
+        font-size: 14px;
+        color: $alpheioscolorvivid;
+        font-weight: normal;
+        padding: 0;
+
+        span {
+            display: inline-block;
+            vertical-align: top;
+            position: relative;
+        }
+
+        .nav-link-current-text-icon svg {
+            fill: $alpheioscolorvivid;
+            width: 15px;
+            height: 15px;
+            vertical-align: top;
+            position: relative;
+            top: 3px;
+        }
+    }
+
+    .navbar-section-ref {
+        display: flex;
+        text-align: center;
+        padding: 0.5rem 0;
+        
+        .nav-link {
```

Comment:
as above for the title, can we use variables here? Style guide defines small label as 12px regular sans and then says that this text should be small label on Mobile and body on Desktop. So I think we need a new $label-large variable, and then we could have the base style for mobile be : font-size: $label-small; font-family: $font-family-sans; and override for media-breakpoint-up-lg to font-size: $base-font-size; font-family $font-family-sans